Mathlib/Data/Nat/Multiplicity.lean

```diff
@@ -26,7 +26,8 @@ coefficients.
 ## Multiplicity calculations
 
 * `Nat.Prime.multiplicity_factorial`: Legendre's Theorem. The multiplicity of `p` in `n!` is
-  `n/p + ... + n/p^b` for any `b` such that `n/p^(b + 1) = 0`.
+  `n/p + ... + n/p^b` for any `b` such that `n/p^(b + 1) = 0`. See `padicValNat_factorial` for the
+  for the corresponding theorem on `p`-adic valuations.
```

Comment:
```suggestion
  for this result stated in the language of `p`-adic valuations.
```

Reply:
Great! I also edited the similar sentence in the `padicValNat` file.